plugins/woocommerce-admin/client/marketplace/components/category-selector/category-selector.tsx

```diff
@@ -104,10 +104,13 @@ export default function CategorySelector(): JSX.Element {
 
 	if ( isLoading ) {
 		return (
-			<div className="woocommerce-marketplace__category-selector-loading">
-				<p>{ __( 'Loading categories…', 'woocommerce' ) }</p>
-				<Spinner />
-			</div>
+			<ul className="woocommerce-marketplace__category-selector">
+				{[...Array(5)].map(() =>
+					<li className="woocommerce-marketplace__category-item">
+						<CategoryLink slug="" label="" selected={ false } />
```

Comment:
What if we had a dedicated loading prop?
This works ok, but it's not clear how to start the loader.
Definitely not a blocker.